src/components/plots/helpers/populateHeatmapData.js

```diff
@@ -121,9 +142,11 @@ const populateHeatmapData = (
     // we want to avoid displaying elements which are not in a louvain cluster
     // so initially consider as enabled only cells in louvain clusters
     // See: https://biomage.atlassian.net/browse/BIOMAGE-809
-    const louvainClusters = hierarchy.find((clusters) => clusters.key === 'louvain');
-
-    const cellIsInLouvainCluster = getCellsSetInGroup(louvainClusters);
+    const selectedCellSet = heatmapSettings?.selectedCellSet ? heatmapSettings.selectedCellSet : 'louvain';
```

Comment:
I don't understand the addition of line 145, can you explain? Also, is it possible to remove the hardcoding of `louvain` in here? In the very hear future, we will want to support other types of clustering, not only louvain, and this line will be a problem

Reply:
This is for the 'Select data' menu, when we don't have that option in the config - use 'louvain', don't know how can that be not hardcoded at the moment.

Reply:
fair enough, let's worry about it later then